### PR TITLE
Log exception info in auto_tick

### PR DESCRIPTION
**Pull request by justcalamari**

### CODE REVIEW

File: conda_forge_tick/auto_tick.xsh [hunk 1]
```diff
@@ -186,8 +186,7 @@ def main(args=None):
                     if is_github_api_limit_reached(e, gh):
                         break
             except Exception as e:
-                logger.critical('NON GITHUB ERROR')
-                logger.critical(str(e))
+                logger.exception('NON GITHUB ERROR')
```
Comment (scopatz) [inline conda_forge_tick/auto_tick.xsh]: aren't you missing the actual error message here?

Comment (justcalamari) [reply conda_forge_tick/auto_tick.xsh]: My understanding of the `exception` method is that it includes the traceback in the logger message.

Comment (CJ-Wright) [reply conda_forge_tick/auto_tick.xsh]: Will this also include the standard error for CLI tools?

Comment (scopatz) [reply conda_forge_tick/auto_tick.xsh]: Oh you are right!

**Comment by CJ-Wright** [comment]
@justcalamari would it be possible to add the standard error to this? I don't expect the `update-cb3` command to report it's traceback inside the xonsh process so this won't capture it. I guess we could use the python side and not the CLI?

**Pull request merged**